client/components/nux-notice/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
```

Comment:
Getting `PropTypes` from react directly will be deprecated in React 16. For new components, we should start using the prop-types package from npm.